eng/DotNetBuild.props

```diff
@@ -88,7 +88,7 @@
 
       <!-- Handle system libraries -->
       <UseSystemLibs Condition="'$(UseSystemLibs)' != ''">+$(UseSystemLibs)+</UseSystemLibs>
-      <InnerBuildArgs Condition="'$(PortableBuild)' != 'true'">$(InnerBuildArgs) --cmakeargs -DCLR_CMAKE_USE_SYSTEM_BROTLI=true</InnerBuildArgs>
+      <InnerBuildArgs Condition="$(UseSystemLibs.Contains('+brotli+'))">$(InnerBuildArgs) --cmakeargs -DCLR_CMAKE_USE_SYSTEM_BROTLI=true</InnerBuildArgs>
```

Comment:
`UseSystemLibs` is used in source-build for our distro partners to build against system brotli.

I don't expect issues with armv6 like zlib-ng as brotli isn't accelerated like zlib-ng is on various platforms.

Reply:
AFAIK, it's not set automatically anywhere. Even in source-build, we are manually passing it at build-time when we want to build the full .NET SDK for a platform. The only exception is for 9.0 (only) where we assume system brotli (https://github.com/dotnet/runtime/pull/107225 ).

Reply:
It would be nice if we can make the vmr default to the configuration that is preferred by most distro maintainers.

Reply:
> It would be nice if we can make the vmr default to the configuration that is preferred by most distro maintainers.

I've created an issue for this: https://github.com/dotnet/source-build/issues/4625.